libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/view/MapboxRouteOverviewButton.kt

```diff
@@ -88,13 +89,14 @@ class MapboxRouteOverviewButton : ConstraintLayout {
     fun showTextAndExtend(duration: Long) {
         if (!isAnimationRunning) {
             isAnimationRunning = true
-            val extendToWidth = EXTEND_TO_WIDTH * context.resources.displayMetrics.density
+            val text = context.getString(R.string.mapbox_route_overview)
+            val extendToWidth = (binding.routeOverviewText.measureText(text) + textWidth)
```

Comment:
what's the difference between `textWidth` and `...measureText(text)`?

Reply:
`textWidth` is actually width of the text view when it is shrunk and no text is shown. 
I renamed the property to reflect that.